### PR TITLE
Make protobuf_benchmarks/ have a single benchmarking program

### PR DESCRIPTION
**Pull request by mkustermann**
This makes the protobuf_benchmarks/ use a single benchmarking program -
shared between VM and Dart2js.

It uses conditional imports to decide how to read file contents (which
is different from VM and Dart2js).

This is already done in query_benchmarks, though using the deprecated
`dart:js` method.

=> We align this between query_benchmarks and protobuf_benchmarks.
=> We use the `package:js/js.dart` approach (since `dart:js` is
   deprecated)

### CODE REVIEW

File: protobuf_benchmarks/lib/readfile.dart [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
```
Comment (mkustermann) [inline protobuf_benchmarks/lib/readfile.dart:1]: It's 2018 - since it's a copy of the one from `query_benchmark`.

**Review by osa1 — APPROVED** [review]
LGTM. If you could fix the formatting I'll merge.

**Comment by mkustermann** [comment]
@osa1 Should be fixed.

**Pull request merged**